modules/azurerm/AKS-Generic/aks_cluster.tf

```diff
@@ -42,13 +41,13 @@ resource "azurerm_kubernetes_cluster" "aks_cluster" {
     zones                        = var.default_node_pool_availability_zones
     os_disk_size_gb              = var.default_node_pool_os_disk_size_gb
     os_disk_type                 = var.default_node_pool_os_disk_type
-    enable_auto_scaling          = var.default_node_pool_enable_auto_scaling
+    auto_scaling_enabled         = var.default_node_pool_enable_auto_scaling
     vnet_subnet_id               = azurerm_subnet.aks_node_pool_subnet.id
     max_count                    = var.default_node_pool_max_count
     min_count                    = var.default_node_pool_min_count
     max_pods                     = var.default_node_pool_max_pods
     orchestrator_version         = var.default_node_pool_orchestrator_version
-    enable_node_public_ip        = false
+    node_public_ip_enabled       = false
```

Comment:
Shall we assign this to a variable and set the default value to false?

Reply:
Sure, Will do that @ayeshajay.